docs/changelog.md

```diff
@@ -12,6 +12,9 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Config.kt)
 
+* **feature-downloads**
+  * Fixing bug #2265. The download notification shows "Unsuccessful download".
```

Comment:
I think this could be a bit more descriptive: What was the issue that was fixed?

Reply:
👍

Reply:
Is this better ?
* **feature-downloads**
  * Fixing bug #2265. In some occasions, when trying to download a file, the download failed and the download notification shows "Unsuccessful download".

Reply:
yep 👍

Reply:
:)